crates/services/consensus_module/poa/src/service_test/manually_produce_tests.rs

```diff
@@ -42,7 +58,7 @@ async fn can_manually_produce_block(
     ctx_builder.with_txpool(txpool);
 
     let mut importer = MockBlockImporter::default();
-    let (tx, mut rx) = tokio::sync::mpsc::channel(times.len() + 1);
+    let (tx, mut rx) = tokio::sync::mpsc::channel(times.len());
```

Comment:
Why did channel length decrease here?

Reply:
Because there is no reason to keep it +1 =D